client/src/components/Sidebar/menu/SubMenuItem.scss

```diff
@@ -30,25 +30,15 @@
 }
 
 .sidebar-sub-menu-panel {
-  display: flex;
-  flex-direction: column;
-  background: $nav-submenu-bg;
-  width: $menu-width;
-  height: 100vh;
+  @apply w-flex w-flex-col w-bg-primary-200 w-w-sidebar w-h-screen;
 
   > h2,
   &__list {
     width: $menu-width;
   }
 
   > h2 {
-    display: block;
-    padding: 0.2em 0;
-    font-size: 1.2em;
-    font-weight: 500;
-    text-transform: none;
-    text-align: center;
-    color: $color-menu-text;
+    @apply w-min-h-[160px] w-inline-flex w-flex-col w-justify-center w-items-center w-text-center w-font-medium w-text-white/80 w-mb-0 w-mt-[35px] w-normal-case w-text-18;
```

Comment:
Okay great points! I've adjusted this based on your feedback
- I've used `w-h4` because it's closer to the size we need. (Not sure if this should then be a <h4> or if we just make it a `<span>`.
- I've added a comment explaining what those arbitrary values are for and rearranged order. 
- I've change the text to be white because the contrast helps it look more like a title 
- Removed unused normal-case and text-center classes.

Also I believe there might be new designs in the future that will focus more on these panels as well as the page explorer.